index.html

```diff
@@ -167,7 +167,8 @@ <h3 class="to-animate">
 					<h2 class="to-animate">SCHEDULE</h2>
 					<div class="row">
 						<div class="col-md-8 col-md-offset-2 subtext to-animate">
-							<h3>Schedule TBD</h3>
+							<h3>Schedule TBD.</h3>
+                            <h3>Click <a href="https://geohackweek.github.io/ghw2016/">here</a> to view the schedule of tutorials from our 2016 Geo Hackweek.</h3>
```

Comment:
we don't currently have any link to 2016 material. I'm thinking people will want to see that in deciding whether to apply.